### PR TITLE
Bump incremental version after adding allocation routines.

### PR DESCRIPTION
**Pull request by sjindel-google**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: ffi
-version: 0.1.1+1
```
Comment (mkustermann) [inline pubspec.yaml]: I missed this in my review: We need to have `CHANGELOG.md` entries for the changes.

Comment (sjindel-google) [reply pubspec.yaml]: Done.

**Comment by sjindel-google** [comment]
> Please also add a change log.

Done.

**Comment by mit-mit** [comment]
Also, were those changes actually compatible with Dart 2.5.0 ?

**Comment by mit-mit** [comment]
I published a version 0.1.2 that rolls back the allocation changes so that it's compatible with Dart 2.5.0.

For these allocation changes I suggest we raise the SDK constraint to Dart 2.6.0-dev.6.0 *and* that we publish this package to a pub.dev dev channel: https://dart.dev/tools/pub/publishing#publishing-prereleases

**Pull request merged**